articles/spotify-python-part-1/spotipy_spotify_python.md

```diff
@@ -0,0 +1,129 @@
+![img](py_img.jpg)
+
+# Spotify API with Spotipy Part 1
+
+## Introduction 
+
+Spotify is a wonderful music streaming app with powerful API functionality. The API is both beginner-friendly and offers a wide range of endpoints. This makes Spotify API a great way to get started for beginner programmers who are interested in getting involved in web APIs and even data science, or advanced programmers who want more control over their playlists. Spotipy is a popular wrapper for the Spotify API. It enables and supports all the features of the Spotify API but allows them to be designed and run in python. This allows for scalability and also will come in handy when we start dealing with large amounts of data.
```

Comment:
Add a link to Spotipy's website when you first mention Spotipy.

Reply:
fixed